### PR TITLE
feat: track volume

### PR DESCRIPTION
**Pull request by 0xArdy**

### CODE REVIEW

**Review by agusduha — DISMISSED** [review]
`/volume` endpoint is not being used in the front, its only for quey purposes?

**Comment by 0xArdy** [comment]
> `/volume` endpoint is not being used in the front, its only for quey purposes?

that's correct :ok_hand:

File: src/hooks/useTransactions/useCCTP.tsx [hunk 1]
```diff
@@ -28,8 +28,28 @@ export const useCCTP = () => {
       allowance,
       approve,
       data: cctpData,
+      trackVolume: () =>
+        track({
+          address: selectedToken?.address as string,
+          symbol: selectedToken?.symbol as string,
+          decimals: selectedToken?.decimals as number,
+          amount: parseTokenUnits(amount).toString(),
+          chainId: selectedToken?.chainId || 0,
```
Comment (0xShaito) [inline src/hooks/useTransactions/useCCTP.tsx:37]: can this happen? A token without chainID

**Review by 0xShaito — CHANGES_REQUESTED** [review]

File: src/hooks/useTransactions/useCCTP.tsx (hunk 1 above)
Comment (0xArdy) [reply src/hooks/useTransactions/useCCTP.tsx]: fixed https://github.com/defi-wonderland/superchain-bridge-ui/pull/102/commits/dac6adf9a43a23e47ee9958705e2a6e50f70ebfd

**Review by 0xShaito — APPROVED** [review]

**Review by 0xShaito — APPROVED** [review]

**Review by agusduha — APPROVED** [review]

**Pull request merged**